pages/partials/navTabs/instructorCourseAdmin.ejs

```diff
@@ -1,9 +1,9 @@
 <% if (authz_data.has_course_permission_own) { %>
     <%- include('navTab', {
-        activeSubPage:  'access',
-        urlSuffix:      '/course_admin/access',
-        iconClasses:    'far fa-calendar-alt',
-        tabLabel:       'Access'}) %>
+        activeSubPage:  'staff',
+        urlSuffix:      '/course_admin/staff',
+        iconClasses:    'fas fa-users',
+        tabLabel:       'Staff'}) %>
```

Comment:
Nav tabs are typically in alphabetical order, I believe this should go between "Questions" and "Settings".

Reply:
Good catch.